rust_src/src/lib.rs

```diff
@@ -25,6 +26,12 @@ pub use math::Ftimes;
 pub use math::Fmax;
 pub use math::Fmin;
 pub use math::Fquo;
+pub use numbers::Fintegerp;
+pub use numbers::Finteger_or_marker_p;
+pub use numbers::Ffloatp;
+pub use numbers::Fnatnump;
+pub use numbers::Fnumberp;
+pub use numbers::Fnumber_or_marker_p;
```

Comment:
It looks to me like these functions don't need to be exported. I can't see them being used elsewhere in the C.